apps/api-extractor/src/api/IConfigFile.ts

```diff
@@ -63,14 +68,37 @@ export interface IConfigApiReport {
   enabled: boolean;
 
   /**
-   * The filename for the API report files.  It will be combined with `reportFolder` or `reportTempFolder` to produce
-   * a full output filename.
+   * The filename for the "complete" API report file, which will encompass all API items, regardless of release tag.
    *
    * @remarks
-   * The file extension should be ".api.md", and the string should not contain a path separator such as `\` or `/`.
+   * It will be combined with {@link IConfigApiReport.reportFolder} and {@link IConfigApiReport.reportTempFolder} to
```

Comment:
Updated the docs here to reflect the new contract. The contract itself was updated to preserve backwards compatibility, while also offering a policy that is easy to reason about.